### DISCUSSION TITLE
Option to change Created by field

### DISCUSSION THREAD

**Discussion by kilim1983** [YetiForceCompany/YetiForceCRM#17576] [How to] [answered]
Hello,

I want to know if there is option to change Created by field for example in Tasks?

![image](https://github.com/user-attachments/assets/1b9e9a80-0ffb-45ec-8ce9-729fac443a97)

Regards,
Damian Krawczyk

**Comment by rskrzypczak** [comment]
as for the label, you can do it in two ways:
- in the field editing panel (`Software configuration/Standard modules/Edit fields`)
- in the language panel, by simply changing the translation (`Software configuration/System tools/Language Management`)

**Reply by kilim1983** [reply]
Hello,

I probably didn't describe precisely what I mean. I mean the situation in which I create a task. The "Created by/Utworzył" field contains my data (Damian Krawczyk). When editing a record, this field is not editable. Can such editing be enabled? I want to have option to change this field.
Regards,
Damian Krawczyk

**Comment by rskrzypczak** [comment, marked as answer]
Of course, there is no such option, because it would allow for entering false data. 
This is a system field, unavailable for editing.
